src/Frontend/src/components/PageHeader.vue

```diff
@@ -13,13 +12,17 @@ import DashboardMenuItem from "@/components/dashboard/DashboardMenuItem.vue";
 import FeedbackButton from "@/components/FeedbackButton.vue";
 import ThroughputMenuItem from "@/views/throughputreport/ThroughputMenuItem.vue";
 import AuditMenuItem from "./audit/AuditMenuItem.vue";
+import { useServiceControlStore } from "@/stores/ServiceControlStore";
+import { storeToRefs } from "pinia";
 
+const serviceControlStore = useServiceControlStore();
+const { isMonitoringEnabled } = storeToRefs(serviceControlStore);
 // prettier-ignore
 const menuItems = computed(
   () => [
   DashboardMenuItem,
   HeartbeatsMenuItem,
-  ...(isMonitoringEnabled() ? [MonitoringMenuItem] : []),
+  ...(isMonitoringEnabled ? [MonitoringMenuItem] : []),
```

Comment:
Does this need `.value`?
```suggestion
  ...(isMonitoringEnabled.value ? [MonitoringMenuItem] : []),
```